bridge/bridgeapp/templates/category.html

```diff
@@ -1,5 +1,5 @@
 {% extends 'home.html' %} {% block main %}
-<h2>{{ category.type }}</h2>
+<h2><u>{{ category.type }}</u></h2>
```

Comment:
As to the <u> tag, all HTML/CSS literature would suggest text decoration or emphases be done via CSS, not HTML. Besides, this is a style done prematurely and subject to change in my opinion.